### DISCUSSION TITLE
Unstable german translation in context "Review"

### DISCUSSION THREAD

**Discussion by JanEmde** [OP-TED/eForms-SDK#1005] [Q&A] [answered]
The german translations for the groups and fields (names and descriptons!) with regard to "review" are very unstable. The fields and groups alternate between "Überprüfung" and "Nachprüfung", although the same thing is meant in each case. Sometimes even the description and name of a single BT differs.

As far as I know, only the term "Nachprüfung" resp. "Nachprüfungsverfahren" is commonly used in German-speaking public procurement law. Therefore, my suggestion would be to use these terms in all the fields and groups mentioned.

**Comment by YvesJo** [comment, marked as answer]
Hi,
Thanks for your input.
Please keep in mind that the review section is not yet fully finalized and ma be subject to changes and so do the labels.
Your recommendations will be shared internally in order to have them into consideration when time will come to deal with Review labels.
KR

**Reply by JanEmde** [reply]
@YvesJo Any news?
